File: MetricsCalculators/test/cpp/Person.cpp
#include "Person.h"

Person::Person(unsigned int id, string name)
{
	this->id = id;
	this->name = name;
}

string Person::getName() const{
	return name;
}

unsigned int Person::getId() const{
	return id;
}

void Person::setName(string name){
	this->name = name;
}

bool Person::compareTo(const Person &person){
	return this->name < person.name;
}

File: MetricsCalculators/test/java/src/main/java/store/Person.java
package store;

public class Person {

	int id;
	String name;
	
	public Person(int id, String name) {
		this.id = id;
		this.name = name;
	}
	public int getId() {
		return id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	
	public boolean compareTo(Person person) {
		return name.compareTo(person.name) < 0;
	}

}
